src/App.js

```diff
@@ -9,7 +11,14 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <div className="App">
+    <Article
+      title={article.title}
+      // eslint-disable-next-line no-unused-vars
+      text={article.text}
+      date={article.date}
```

Comment:
```suggestion
      {...article}
```
It's correct, but here is another way to do it

https://medium.com/@kevinyckim33/jsx-spread-operator-component-props-meaning-3c9bcadd2493